front/src/modules/favorites/hooks/useFavorites.ts

```diff
@@ -43,6 +47,36 @@ export const useFavorites = () => {
     });
   };
 
+  const updateFavoritesOrder = async (
+    favorites: GetFavoritesQuery['findFavorites'],
```

Comment:
we avoid passing GraphQL query types in components / hooks. This is because the updateFavoritesOrder function should not have to know about the Get query itself.
Instead, let's be very precise and only specify what this function needs to perform its logic. To do that you can do: Pick<Favorite, 'id' | 'index'> for example